SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Uniswap Labs Security
```

Comment:
The audit also says
```
Consider adding a NatSpec comment containing a security contact above each contract definition. Using the @custom:security-contact convention is recommended
```

Shall we add the contact at the top of the PoolManager too?

Reply:
I'm not against it but I'm also fine with just the security file

Reply:
ok fair i'll merge